src/gwt/src/org/rstudio/studio/client/workbench/views/buildtools/BuildClearButton.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * ConsoleClearButton.java
+ *
+ * Copyright (C) 2020 by RStudio, PBC
+ *
+ * Unless you have received this program directly from RStudio pursuant
+ * to the terms of a commercial license agreement with RStudio, then
+ * this program is licensed to you under the terms of version 3 of the
+ * GNU Affero General Public License. This program is distributed WITHOUT
+ * ANY EXPRESS OR IMPLIED WARRANTY, INCLUDING THOSE OF NON-INFRINGEMENT,
+ * MERCHANTABILITY OR FITNESS FOR A PARTICULAR PURPOSE. Please refer to the
+ * AGPL (http://www.gnu.org/licenses/agpl-3.0.txt) for more details.
+ *
+ */
+package org.rstudio.studio.client.workbench.views.buildtools;
+
+import com.google.gwt.dom.client.Style.Position;
+import com.google.gwt.resources.client.ImageResource;
+import com.google.gwt.user.client.ui.Composite;
+import com.google.gwt.user.client.ui.SimplePanel;
+import com.google.inject.Inject;
+import org.rstudio.core.client.command.AppCommand;
+import org.rstudio.core.client.theme.res.ThemeStyles;
+import org.rstudio.core.client.widget.ToolbarButton;
+import org.rstudio.studio.client.workbench.commands.Commands;
+
+public class BuildClearButton extends Composite
```

Comment:
Do you need a custom class for this? The terminal pane has a clear button, and it just does it with a straight ToolbarButton like this:

```
      clearButton_ = commands_.clearTerminalScrollbackBuffer().createToolbarButton();
      clearButton_.addStyleName(ThemeStyles.INSTANCE.terminalClearButton());
      toolbar.addRightWidget(clearButton_);
```

Reply:
Due to the way the "mouse down" action happens I believe that is why the custom class was created for the ConsoleClear button as well. Not getting a TON of value out of having its own class but it's following the precedent set with that icon elsewhere.